src/components/ControlPanel/ControlPanelBox/ControlPanelBox.js

```diff
@@ -20,12 +23,28 @@ export const ControlPanelBox = ({ box, disabled = false, hidden = false }) => {
             <S.Text>{_(box.iconName)}</S.Text>
           </S.DisabledLink>
         ) : (
-          <S.Link target="_self" className="dp-white" href={box.linkUrl}>
-            <div>
-              <S.Image src={box.imgSrc} alt={_(box.imgAlt)} />
-            </div>
-            <S.Text>{_(box.iconName)}</S.Text>
-          </S.Link>
+          <>
+            {box.status ? (
+              <S.StatusImage
+                src={
+                  box.status === 'connected'
+                    ? connected
+                    : box.status === 'alert'
+                    ? connection_alert
+                    : disconnected
+                }
+                alt=""
```

Comment:
What do you think about using the status as `alt`?

```suggestion
                alt={box.status}
```